dev/com.ibm.ws.org.apache.cxf.jaxrs/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove this file

Reply:
ugh stupid eclipse re-creating deleted files!